suppression-list/csv2optout.py

```diff
@@ -0,0 +1,208 @@
+import csv
+import logging
+import re
+import sys
+from datetime import datetime, timezone
+from pathlib import Path
+
+import click
+
+logger = logging.getLogger(__name__)
+
+
+SQL_COMMANDS_PRE = """
+CREATE OR REPLACE PROCEDURE raise_notice (s TEXT) LANGUAGE plpgsql AS
+$$
+BEGIN
+    RAISE NOTICE '%', s;
+END;
+$$;
+
+CREATE TEMPORARY TABLE IF NOT EXISTS imported_{tmp_suffix} (
```

Comment:
Why do we need a table suffix for a temp table? The table would be dropped at the end of the session by default (or at the end of a transaction if we specify `on commit drop`)

Reply:
Yes, you're right, this is not necessary